### PR TITLE
Scroll of Harvests 翻译提交

### PR DESCRIPTION
**Pull request by ChuijkYahus**

### CODE REVIEW

**Comment by ChuijkYahus** [comment]
/mv-recursive "projects/1.18/assets/scroll-of-harvests/harvest/" "projects/1.18/assets/scroll-of-harvests/harvests/"
/mv-recursive "projects/1.19/assets/scroll-of-harvests/harvest/" "projects/1.19/assets/scroll-of-harvests/harvests/"
/mv-recursive "projects/1.20/assets/scroll-of-harvests/harvest/" "projects/1.20/assets/scroll-of-harvests/harvests/"

**Comment by ChuijkYahus** [comment]
此模组1.18.2最新版为1.3.0

File: projects/1.18/assets/scroll-of-harvests/harvests/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
+{
+    "harvests.tier": "等级",
+    "harvests.tier.tooltip": "各等级根据相对关系排列，且可能因模组不同而改变顺序，悬停于等级名以获取更多信息",
+    "harvests.material": "材料",
+    "harvests.material.tooltip": "修复该等级工具所需的材料，通常这些材料也用于该等级工具的合成配方",
```
Comment (Cactusstudent) [inline projects/1.18/assets/scroll-of-harvests/harvests/lang/zh_cn.json:5]: 考虑吧“该等级”改为“该品质”吗？

Comment (ChuijkYahus) [reply projects/1.18/assets/scroll-of-harvests/harvests/lang/zh_cn.json:5]: 那样需要把全文的几乎所有等级一词全部换成品质
也就是Tier这个词的翻译

Comment (Cactusstudent) [reply projects/1.18/assets/scroll-of-harvests/harvests/lang/zh_cn.json:5]: 建议换，毕竟这属于原版用词了

**Review by Cactusstudent — CHANGES_REQUESTED** [review]

**Review by Cactusstudent — APPROVED** [review]

**Pull request merged**